lib/jcsda-emc/spack-stack/stack/cmd/stack_cmds/create.py

```diff
@@ -51,44 +51,80 @@ def container_config_help():
     help_string = "Pre-configured container." + os.linesep
     help_string += "Available options are: " + os.linesep
     for config in container_configs:
+        if config == "README.md":
+            continue
         help_string += "\t" + config.rstrip(".yaml") + os.linesep
     return help_string
 
 
+def container_specs_help():
```

Comment:
The diffs in this file are confusing. It may be better checking out the current code and this PR and verify side by side in meld (which sometimes does a better job with diffs). Most of the common arguments on the left moved down into the env section, and ctr got its own arguments.